WTWSMS/common/minor_titles/00_minor_titles.txt

```diff
@@ -691,11 +693,6 @@ title_despot = {
 			has_landed_title = e_byzantium
 			has_landed_title = e_roman_empire
 		}
-		OR = {
-			religion_group = christian
-			religion_group = jewish_group
-			religion_group = pagan_group
-		}
```

Comment:
You misunderstand. While it makes perfect sense that a Hindu or non-Latin/non-Hellenic character should not restore the Roman Empire, if a character of a foreign religion or culture USURPS the Roman throne, he should be able to use Roman features.

In other words, if a Hindu is somehow installed as Roman Emperor, he should be able to use the privileges and powers of a Roman Emperor. To do otherwise would be nonsensical, but allowing that same Hindu to _recreate_ the Roman Empire would also be nonsensical. I struck a fair balance.